src/openforms/registrations/contrib/objects_api/tests/test_backend.py

```diff
@@ -859,7 +859,7 @@ def test_submission_with_objects_api_backend_bsn(self, m):
             document_create_body["informatieobjecttype"],
             "https://catalogi.nl/api/v1/informatieobjecttypen/1",
         )
-        self.assertNotIn("vertrouwelijkheidaanduiding", document_create_body)
+        self.assertNotIn("document_vertrouwelijkheidaanduiding", document_create_body)
```

Comment:
there is no reason why this should ever be in the document create body, it's like asserting that `captain_picard` is not in `document_create_body`

I think this test asserts that if there's no explicit VA specified, that it should not be present in the request body. If a global value is configured, then it should be asserted that this global value is used.

Normally the VA is derived from the zaaktype/documenttype if not specified explicitly, so it should definitely still be possible to not specify any default value and have this key be absent in the document create body so that the API can derive it from the type-information instead.